requirements.txt

```
@@ -1,11 +1,3 @@
-google
-beautifulsoup4
-biopython
-aiohttp
-pandas
-pandas
-opencv-python
-lxml
-math
-typing
-requests
+numpy
```

Нужно указывать точные версии, например
```suggestion
numpy==1.21.1
```